atomic_reactor/util.py

```diff
@@ -985,7 +985,95 @@ def get_manifest_list(image, registry, insecure=False, dockercfg_path=None):
     return response
 
 
-def get_config_from_registry(image, registry, digest, insecure=False,
+def get_all_manifest_digests(image, registry, insecure=False, dockercfg_path=None,
```

Comment:
Would `get_all_manifests` be a better name for this function? I think it returns the manifests themselves, and not the digests.

Reply:
ok